walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.trim().replaceAll("[\\n\\t]", "");
```

Comment:
I think you don't need the `replaceAll`, `trim` already takes care of tabs and newlines.

Reply:
Oh, I thought trim was only spaces. I will remove the replaceAll().